src/vaults/BalanceSheet.sol

```diff
@@ -39,8 +39,13 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
     ISharePriceProvider public sharePriceProvider;
 
     mapping(PoolId => mapping(ShareClassId => mapping(address => bool))) public manager;
-    mapping(PoolId poolId => mapping(ShareClassId scId => int128 amount)) public queuedShares;
-    mapping(PoolId poolId => mapping(ShareClassId scId => mapping(AssetId assetId => int128))) public queuedAssets;
+
+    mapping(PoolId poolId => mapping(ShareClassId scId => uint128 amount)) public queuedShareIssuances;
+    mapping(PoolId poolId => mapping(ShareClassId scId => uint128 amount)) public queuedShareRevocations;
+    mapping(PoolId poolId => mapping(ShareClassId scId => mapping(AssetId assetId => uint128))) public
```

Comment:
One styling nit, in this entire contract: could we always put deposit above withdraw, and issue above revoke?

The order is now sometimes swapped and that is a bit confusing to read/audit.